File: python/Problem009.py
# Problem 9
# By: Tim Oram
#
# Runtime: 4.560s 

import math
import sys

for i in xrange(2, 1001):
	for j in xrange(i+1,1001 - i):
		for k in xrange(j+1,1001 - i - j):
			if (i+j+k) != 1000:
				continue
			
			if (math.pow(i,2) + math.pow(j,2)) == math.pow(k,2):
				print "Answer:", i * j * k
				sys.exit()

File: c/Problem009.c
// Problem 9
// By: Tim Oram

#include <stdio.h>

int main (void) {

	int i, j, k;
	for (i = 1; i <= 997; ++i) {
		for (j = i+1; j <= 999; ++j) {
			for (k = j+1; k <= 998; ++k) {
				if (i+j+k == 1000 && i*i + j*j == k*k) {
					printf("Answer: %d\n", i*j*k);
					return 0;
				}
			}
		}
	}
	return 1;
}
